tests/operators/hive_operator.py

```diff
@@ -92,38 +96,54 @@ class HiveOperatorTest(HiveEnvironmentTest):
 
     def test_hiveconf_jinja_translate(self):
         hql = "SELECT ${num_col} FROM ${hiveconf:table};"
-        t = operators.hive_operator.HiveOperator(
+        t = HiveOperator(
             hiveconf_jinja_translate=True,
             task_id='dry_run_basic_hql', hql=hql, dag=self.dag)
         t.prepare_template()
         self.assertEqual(t.hql, "SELECT {{ num_col }} FROM {{ table }};")
 
     def test_hiveconf(self):
         hql = "SELECT * FROM ${hiveconf:table} PARTITION (${hiveconf:day});"
-        t = operators.hive_operator.HiveOperator(
+        t = HiveOperator(
             hiveconfs={'table': 'static_babynames', 'day': '{{ ds }}'},
             task_id='dry_run_basic_hql', hql=hql, dag=self.dag)
         t.prepare_template()
         self.assertEqual(
             t.hql,
             "SELECT * FROM ${hiveconf:table} PARTITION (${hiveconf:day});")
 
+    @mock.patch('airflow.operators.hive_operator.HiveOperator.get_hook')
+    def test_mapred_job_name(self, mock_get_hook):
+        mock_hook = mock.MagicMock()
+        mock_get_hook.return_value = mock_hook
+        t = HiveOperator(
+            task_id='test_mapred_job_name',
+            hql=self.hql,
+            dag=self.dag)
 
-if 'AIRFLOW_RUNALL_TESTS' in os.environ:
+        fake_execution_date = timezone.datetime(2018, 6, 19)
+        fake_ti = TaskInstance(task=t, execution_date=fake_execution_date)
+        fake_context = {'ti': fake_ti}
+
+        t.execute(fake_context)
+        self.assertEqual(
+            "{}.{}.{}".format(self.dag.dag_id, t.task_id,
+                              fake_execution_date.isoformat()), mock_hook.mapred_job_name)
 
-    import airflow.hooks.hive_hooks
-    import airflow.operators.presto_to_mysql
+
+if 'AIRFLOW_RUNALL_TESTS' in os.environ:
```

Comment:
Why is this? I would like to get rid of these exclusions.

Reply:
I don't have context on why there's such check. This line is just result of auto import optimization from my IDE. Let me change this back. We can start another PR to remove such exceptions.